docs/application/dotnet/guides/app-management/app-controls.md

```diff
@@ -145,6 +145,7 @@ appcontrol.ApplicationId = "org.tizen.TestApp";
 
 AppControl.SendLaunchRequest(appcontrol);
 ```
+If you want to request launching an AppControl asynchronously, you can use SendLaunchRequestAsync().
```

Comment:
Suggestion: To make it consistent with the rest of the content in this page and also to avoid wordiness, suggest rephrasing to:
To launch an AppControl asynchronously, you can use SendLaunchRequestAsync().

Reply:
done

Reply:
@[surbhig12](/surbhig12)
SRI-B members tend to change most of sentences to start "To ..." nowadays.
I think a clause is better than a phrase if the sentence is not an item of list.
Please see native/guides/security/privilege.md and location-sensors/device-sensors.md.

Reply:
@journey2w,
I have gone through the suggested references.
I agree with your point that “a clause is better than a phrase if the sentence is not an item of list.” However, my intention to modify this content to start with “To…” is to maintain the consistency and tone of this page and other sections in this page.

Reply:
@surbhig12 
My suggested sentence doesn't spoil the consistency.
To maintain the consistency, we should not always write all sentences  as "To ..." . 
I believe you are a native speaker and a good reviewer. However, it would be great if you rekon context the paragraph and recommend better sentences.